File: Java/TextConverter/src/test/java/tddmicroexercises/textconvertor/HtmlPagesConverterTest.java
package tddmicroexercises.textconvertor;

import static org.junit.jupiter.api.Assertions.assertEquals;

import java.io.IOException;

import org.junit.jupiter.api.Test;

public class HtmlPagesConverterTest {

    @Test
    public void foo() throws IOException {
        HtmlPagesConverter converter = new HtmlPagesConverter("foo");
        assertEquals("fixme", converter.getFilename());
    }

}

File: kotlin/TextConverter/src/test/kotlin/tddmicroexercises/textconvertor/HtmlPagesConverterTest.kt
package tddmicroexercises.textconvertor

import java.io.IOException

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class HtmlPagesConverterTest {
    @Test
    @Throws(IOException::class)
    fun foo() {
        val converter = HtmlPagesConverter("foo")
        assertEquals("fixme", converter.filename)
    }
}
